src/resources/OnTimePrediction.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Amadeus\Resources;
+
+/**
+ * An OnTimePrediction object as returned by the Airport On-Time Prediction API.
+ * @see OnTimePrediction
```

Comment:
the leading resource should be redirected to the endpoint which will use this resource, thus it should be ```@see OnTime```